lib/validation/rules/maxTotalFileSize.ts

```diff
@@ -1,16 +1,15 @@
 import { helpers } from '@vuelidate/validators'
+import { createRule } from '../Rule'
 import { maxTotalFileSize as baseMaxTotalFileSize } from '../validators/maxTotalFileSize'
 
+export const message = {
+  en: (size: string) => `The total file size must be smaller than or equal to ${size}.`,
+  ja: (size: string) => `合計ファイルサイズは最大${size}までです。`
+}
+
 export function maxTotalFileSize(size: string, msg?: string) {
-  return helpers.withParams(
-    { size },
-    helpers.withMessage(
-      ({ $params }) => {
-        return msg ?? `The total file size must be smaller than ${$params.size}.`
-      },
-      (files: File[]) => {
-        return !helpers.req(files) || baseMaxTotalFileSize(files, size)
-      }
-    )
-  )
+  return createRule({
+    message: ({ lang }) => msg ?? message[lang](size),
+    validation: (value: File[]) => !helpers.req(value) || baseMaxTotalFileSize(value, size)
```

Comment:
`!helpers.req(value)` may not be needed according to `createRule` implementation, but not so sure.

Reply:
Fixed: 5c20e70